htdocs/install/mysql/data/llx_accounting_account_be.sql

```diff
@@ -945,3 +945,432 @@ INSERT INTO llx_accounting_account (entity, rowid, fk_pcg_version, pcg_type, acc
 INSERT INTO llx_accounting_account (entity, rowid, fk_pcg_version, pcg_type, account_number, account_parent, label, active) VALUES (__ENTITY__, 1355, 'PCMN-BASE', 'FINAN', '5', '0', 'Placement de trésorerie et de valeurs disponibles', 1);
 INSERT INTO llx_accounting_account (entity, rowid, fk_pcg_version, pcg_type, account_number, account_parent, label, active) VALUES (__ENTITY__, 1356, 'PCMN-BASE', 'EXPENSE', '6', '0', 'Charges', 1);
 INSERT INTO llx_accounting_account (entity, rowid, fk_pcg_version, pcg_type, account_number, account_parent, label, active) VALUES (__ENTITY__, 1357, 'PCMN-BASE', 'INCOME', '7', '0', 'Produits', 1);
+
+INSERT INTO llx_accounting_account (entity, rowid, fk_pcg_version, pcg_type, account_number, account_parent, label, active) VALUES (__ENTITY__, 200439, 'MAR-VERKORT', 'CAPIT', '1',  '0', 'Eigen vermogen, voorzieningen voor risico''s en kosten en schulden op meer dan één jaar', 1);
```

Comment:
Looks ok. Just one thing to fix.
The ID to use (example here 200439) must be an ID lower than 99999.  The 200 000 is added automatically for belgium to avoid conflict with another country.
The previous plan end at 1357, so i suggest to start the new one for dutch for the new plan at 10000
So you can replace the string  
", 200"  into  ", 10"  
and  ",  '200"  into ",  '10"
So ID will be 10439, 10440, ...